### PR TITLE
Feat/language implement

### PR DESCRIPTION
**Pull request by hansac91**
- created language.ts - table of existing languages
- created LanguageInput.tsx - component based on TechStackInput.tsx
- updated schema.prisma according to the instructions  given in the task on Nerdbord
- updated translations

### CODE REVIEW

File: apps/platform-app/src/app/[locale]/(profile)/(components)/MyProfile/ProfileDetails/ProfileDetails.tsx [hunk 1]
```diff
@@ -28,6 +28,16 @@ const ProfileDetails = async (props: ProfileDetailsProps) => {
               ))}
             </div>
           </div>
+          {/* <div className={styles.techStack}>
```
Comment (Sumick) [inline apps/platform-app/src/app/[locale]/(profile)/(components)/MyProfile/ProfileDetails/ProfileDetails.tsx:31]: **nit:** Should it be commented out?

File: apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/EditProfileForm/EditProfileForm.tsx [hunk 2]
```diff
@@ -116,6 +117,11 @@ const EditProfileForm = () => {
       hourlyRateMin: values.hourlyRateMin,
       hourlyRateMax: values.hourlyRateMax,
       currency: Currency.PLN,
+      language: values.language.map((language) => {
```
Comment (Sumick) [inline apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/EditProfileForm/EditProfileForm.tsx:120]: **nit:** This mapper is duplicated - I think we have a file that consists of such methods. Let's add it there, and reuse it. Check `apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/EditProfileForm/mappers.ts`

File: apps/platform-app/src/backend/profile/profile.service.ts [hunk 3]
```diff
@@ -100,6 +100,15 @@ export async function createUserProfile(
       hourlyRateMin: profileData.hourlyRateMin ?? 0,
       hourlyRateMax: profileData.hourlyRateMax ?? 0,
       currency: Currency.PLN,
+      language: {
+        connectOrCreate: profileData.language.map(
+          (lang) => ({
+            where: { name: lang.name },
+            create: { name: lang.name },
+          }),
+          console.log('profileData', profileData),
```
Comment (Sumick) [inline apps/platform-app/src/backend/profile/profile.service.ts:109]: **nit:** log to remove

File: package.json [hunk 4]
```diff
@@ -18,9 +18,12 @@
...
   },
   "devDependencies": {
-    "husky": "^8.0.0",
     "@commitlint/cli": "^17.6.3",
     "@commitlint/config-conventional": "^17.6.3",
+    "husky": "^8.0.0",
     "lint-staged": "^15.1.0"
+  },
+  "dependencies": {
+    "next-intl": "^3.14.1"
```
Comment (Sumick) [inline package.json:27]: **nit:** we should remove these:

```
 "dependencies": {
    "next-intl": "^3.14.1"
   }
```

File: apps/platform-app/package.json [hunk 5]
```diff
@@ -13,10 +13,13 @@
...
     "vercel-build": "prisma generate && prisma migrate deploy && next build",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "prisma:generate": "prisma generate --schema=$PRISMA_SCHEMA_PATH",
+    "prisma:studio": "prisma studio  --schema=./apps/platform-app/prisma/schema.prisma"
   },
   "prisma": {
-    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
+    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts",
+    "schema": "./apps/platform-app/prisma/schema.prisma"
```
Comment (Sumick) [inline apps/platform-app/package.json:22]: **nit:** We should remove this, as Vercel has troubles with deployment now

Zrzut ekranu 2024-06-11 o 13 24 26

File: apps/platform-app/package.json [hunk 6]
```diff
@@ -13,10 +13,13 @@
     "prisma:seed": "yarn prisma db seed",
     "postinstall": "prisma generate",
     "vercel-build": "prisma generate && prisma migrate deploy && next build",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "prisma:generate": "prisma generate --schema=$PRISMA_SCHEMA_PATH",
```
Comment (Sumick) [inline apps/platform-app/package.json:17]: **question:** Why did you need these?

**Review by Sumick — CHANGES_REQUESTED** [review]
Very good job, implementation looks very solid! Congrats Hania! 🔥 Im leaving few minor things I've noticed + we need to fix deployment to make it possible to run on Vercel. I think it's because of these new scripts you've added in this PR. I think we can try removing them, and it should do the job

File: apps/platform-app/src/app/[locale]/(profile)/(components)/MyProfile/ProfileDetails/ProfileDetails.tsx (hunk 1 above)
Comment (hansac91) [reply apps/platform-app/src/app/[locale]/(profile)/(components)/MyProfile/ProfileDetails/ProfileDetails.tsx:31]: I commented it because there is another task in Nerdbord for this case and I totally forgot to delete it here.  I'll use it in second task.

File: apps/platform-app/package.json (hunk 6 above)
Comment (hansac91) [reply apps/platform-app/package.json:17]: When I used command "npx prisma generate" I received an error that the schema.prisma couldn't be found. So that that was my first try to solve the case of changed location. I fixed it later with new instructions and forgot to remove it.

File: apps/platform-app/src/components/LanguageInput/LanguageInput.tsx [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+  chips = [],
+  onTechSelect,
+  onTechRemove,
+  placeholder,
+  name,
+  label,
+  addImportantIcon,
+  tooltipText,
+}) => {
+  const { errors, handleBlur, touched } = useFormikContext<ProfileFormValues>()
```
Comment (Sumick) [inline apps/platform-app/src/components/LanguageInput/LanguageInput.tsx:45]: **nit:** We should place this component close to the form there its used. It's because it uses `useFormikContext`, and it will be impossible to move it later to `ui-system`

**Review by Sumick — APPROVED** [review]
Good job :) Before merging, please move this newly created LanguageDropdown closer to the forms that use it, as currently it won't be possible to move it to our design system

**Review by Sumick — APPROVED** [review]
Looks good!

**Review by mmcode9407 — APPROVED** [review]
GJ
